src/packages/cli/src/Init.ts

```diff
@@ -87,13 +87,29 @@ export class Init implements Command {
     return new Init()
   }
 
-  // TODO add --provider & --url
   private static help = format(`
-    Setup a new Prisma project
+  Setup a new Prisma project
```

Comment:
Changing indentation will not have an effect on the CLI output?

Reply:
No, and here is the new output 
<img width="638" alt="Screen Shot 2021-06-25 at 19 35 16" src="https://user-images.githubusercontent.com/1328733/123463991-824cc700-d5ec-11eb-81b6-e97a67e36f0f.png">

Reply:
I spotted a misalignment, I'll fix